cmd/minikube/cmd/status.go

```diff
@@ -170,6 +170,12 @@ type BaseState struct {
 	StepDetail string `json:",omitempty"`
 }
 
+// ScheduledStopStatus holds string representation of scheduledStopDuration
+type ScheduledStopStatus struct {
+	InitiatedTime string `json:",omitempty"`
```

Comment:
So I think it would be more useful to provide the user with the time left until shutdown, instead of initiation time and scheduled time. It will save the user some calculation.

We can call this:

```go
type ScheduledStopStatus struct {
   TimeToStop string
}
```